.gitignore

```diff
@@ -1,3 +1,4 @@
 *.zip
 *.smx
 builds
+.atom-build.*
```

Comment:
Can you revert the change to the .gitignore file? I'd prefer to keep any editor specific things out.